DOCS/man/options.rst

```diff
@@ -6198,7 +6198,9 @@ them.
     cie1931
         CIE 1931 RGB (not to be confused with CIE XYZ)
     dci-p3
-        DCI-P3 (Digital Cinema Colorspace), SMPTE RP431-2
+        DCI-P3 (Digital Cinema Colorspace), SMPTE RP431-2, DCI whitepoint
+    display-p3
+        P3-D65, SMPTE EG432-1, Netflix masters use with PQ, all Apple devices
```

Comment:
"Netflix masters use with PQ, all Apple devices" seems unnecessary - no other option is documented this way. Just describe it as it is, i.e. "DCI-P3 with D65 whitepoint"

Reply:
DCI-P3 uses DCI whitepoint, that is the whole point. I documented whitepoints that are not D65.

Reply:
maybe looks wired, it's dci-p3-dci and dci-p3-d65, dci in prefix meaning "Digital Cinema Initiatives" not dci wp.

Reply:
No one uses dci-p3-dci? See wikipedia.

Reply:
nobody argue about **use** dci or not.
i just don't want to see you make two p3 **primary** looks like totally different things in document.

I like haasn's proposal, and move SMPTE EG432-1 to start make styles consistent.

Reply:
>two p3 primary looks like totally different things in document.

But they are totally different due to chromatic adaptation due to a different white point.
Morover if we are talking about ICC profiles we always chromatically adapt values there to D50, so D65 white point becomes D50, primaries are shifted and in case of DCI white point it is not on plankian locus, so not CIE D series illuminant even. Additional field chad (chromatic adaptation is added and in case of DCI actually required).

Reply:
Talk is cheap. Show you the code.
https://github.com/ampas/aces-dev/blob/dev/transforms/ctl/lib/ACESlib.Utilities_Color.ctl#L41-L63

Reply:
Yes, ACES uses D60, which is perfect Series D: 0.32168,  0.33767 and is on plankian locus.

But the DCI is not.

>SMPTE EG432-1 to start make styles consistent

No, H.273 shows 431-2 first.

Reply:
```
        SMPTE EG432-1, SMPTE RP431-2, P3-DCI (Theater)
    display-p3
        SMPTE EG432-1, SMPTE RP431-2, P3-D65 (Display)
```
My opinion here. If you don't accept it, ignore it, it's your pr.